stdlib/src/utils/static_tuple.mojo

```diff
@@ -432,3 +433,36 @@ struct InlineArray[ElementType: CollectionElement, size: Int](Sized):
             An `UnsafePointer` to the underlying array.
         """
         return UnsafePointer(self._array).bitcast[Self.ElementType]()
+
+    @always_inline
+    fn __contains__[
+        T: ComparableCollectionElement
+    ](self: Reference[InlineArray[T, size]], value: Self.ElementType) -> Bool:
+        """Verify if a given value is present in the array.
+
+        ```mojo
+        from utils import InlineArray
+        var x = InlineArray[Int, 3](1,2,3)
+        if 3 in x: print("x contains 3")
+        ```
+
+        Parameters:
+            T: The type of the elements in the array. Must implement the
+              traits `EqualityComparable` and `CollectionElement`.
+
+        Args:
+            value: The value to find.
+
+        Returns:
+            True if the value is contained in the array, False otherwise.
+        """
+        constrained[
+            _type_is_eq[T, Self.ElementType](),
+            "T must be equal to Self.ElementType",
+        ]()
+
+        @unroll
```

Comment:
FYI I removed this internally since `@unroll` is removed now.  We introduce a `@parameter for` that I think we can apply here, but I decoupled it for now so we can test the hot-off-the-presses language feature in a follow-up PR.  I dropped a TODO here so we don't forget.